src/main/java/edu/cornell/kfs/pdp/CUPdpPropertyConstants.java

```diff
@@ -4,6 +4,14 @@
 
     public static final String PAYEE_PRINCIPAL_NAME = "payeePrincipalName";
 
+    public static final class PaymentDetail {
+        public static final String PAYMENT_COUNTRY = "paymentGroup.country";

```

Comment:
Why does PAYMENT_COUNTRY need to be in an internal class, and not just in CUPdpPropertyConstants?

Reply:
I was trying to be consistent with how similarly-named properties were being grouped in the PdpPropertyConstantsClass. If you think this adds too much clutter, then I'll remove this nested class and move the constant to the parent class.